app/views/errors/timeout.html.erb

```diff
@@ -0,0 +1,3 @@
+<h2 class="heading"><%= t("errors.timeout.title") %></h2>
+<h3 class="heading"><%= t("errors.timeout.subtitle") %></h3>
+<p><%= t("errors.timeout.html", contact_support_link: link_to(ts("contact Support"), new_feedback_report_path)) %></p>
```

Comment:
Please prefer [lazy lookup](https://guides.rubyonrails.org/i18n.html#lazy-lookup), so for example `<h2 class="heading"><%= t(".page_heading") %></h2>`.

For the link, like ERB Lint says, the link caption should also be localised. We have a [specific standard](https://github.com/otwcode/otwarchive/wiki/Internationalization-%28i18n%29-Standards#hyperlinks) for hyperlinks that should result in something like `<p><%= t(".description_html", contact_support_link: link_to(t(".contact_support"), new_feedback_report_path)) %></p>`, with the added string in the locale file